### PR TITLE
Add backAround

### PR DESCRIPTION
**Pull request by WarrenKnocker**

### CODE REVIEW

File: WarmUp1/backAround/test/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+so "cat" yields "tcatt". The original string will be length 1 or more.
+
+
+backAround("cat") → "tcatt"
+backAround("Hello") → "oHelloo"
+backAround("a") → "aaa"
+
+*/
+
+describe("backAround", () => {
```
Comment (JustinGuedes) [inline WarmUp1/backAround/test/index.test.ts:16]: Can you please add a test for when the string is empty

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

**Comment by WarrenKnocker** [comment]
Request to merge. Updated function & test
